pyproject.toml

```diff
@@ -17,7 +18,7 @@ dependencies = [] # Add project dependencies here, e.g. ["click", "numpy"]
 dynamic = ["version"]
 license.file = "LICENSE"
 readme = "README.rst"
-requires-python = ">=3.8"
+requires-python = ">=3.7"
```

Comment:
Not directly an issue with this PR, but is there a reason the minimum version was specified here as 3.8 and yet the test matrix only included 3.9+?